docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
 
-linkUp/linkDown traps (deprecated)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are two ways PacketFence can show its captive portal for unknown (or unregistered) devices:
 
-This is the most basic setup and it needs a third VLAN: the MAC detection VLAN. There should be nothing in this VLAN (no DHCP server) and it should not be routed anywhere; it is just an void VLAN.
+ * it can use Web Authentication (or also known as hotspot-style authentication) - this works with numerous equipment vendors
+ * it can use a registration VLAN, where PacketFence provides DHCP services and DNS black-holing services - this works with any equipment vendors
 
-When a host connects to a switch port, the switch sends a linkUp trap to PacketFence. Since it takes some time before the switch learns the MAC address of the newly connected device, PacketFence immediately puts the port in the MAC detection VLAN in which the device will send DHCP requests (with no answer) in order for the switch to learn its MAC address. Then pfsetvlan will send periodical SNMP queries to the switch until the switch learns the MAC of the device. When the MAC address is known, pfsetvlan checks its status (existing ? registered ? any violations ?) in the database and puts the port in the appropriate VLAN. When a device is unplugged, the switch sends a 'linkDown' trap to PacketFence which puts the port into the MAC detection VLAN.
+For our example, we will use Web Authentication, as it is supported by the Cisco Catalyst 2960. For more information on various enforcement modes, please refer to the 'Supported Enforcement Modes' sections of this document.
 
-When a computer boots, the initialization of the NIC generates several link status changes. And every time the switch sends a linkUp and a linkDown trap to PacketFence. Since PacketFence has to act on each of these traps, this generates unfortunately some unnecessary load on pfsetvlan. In order to optimize the trap treatment, PacketFence stops every thread for a 'linkUp trap' when it receives a 'linkDown' trap on the same port. But using only linkUp/linkDown traps is not the most scalable option. For example in case of power failure, if hundreds of computers boot at the same time, PacketFence would receive a lot of traps almost instantly and this could result in network connection latency.
+Creating Authentication Source for Guests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-MAC notification traps
-^^^^^^^^^^^^^^^^^^^^^^
+To keep our example simple, we will simply create a captive portal for guests where they will only have to accept the terms and conditions prior gaining network access. To do so, we must first create a 'Null' authentication source. From 'Configuration->Policies and Access Control->Authentication Sources', click on 'Add Source->External->Null'. As 'Name' and 'Description', specify 'null-source'. Then add an 'Authentication Rules' with name 'catchall' with no condition and with the following 'Actions':
```

Comment:
I'd reuse the existing 'null' source instead but that is a matter of taste

Reply:
Why do we pre-create some external sources? I don't think we should.

Reply:
I think that dates back in PF v4 because in v3 SMS and email would work right out of the box so they were left there for easing the upgrades.

Same as the legacy source of /usr/local/pf/conf/admin.conf

IMO, its not a bad idea to have those pre-created since its easier to get things configured quickly.
But, that makes it so the user doesn't really understand some magic.

Both have +/-, I'd say we state on whether or not they stay and make the documentation aligned with our choice so that the user doesn't end up with useless sources

Reply:
I say we axe them. That's my opinion though, but let's bring it to a R&D meeting.